docs/user/site.rst

```diff
@@ -387,6 +387,19 @@ mesh_vpn
         brokers = {'vpn1.alpha-centauri.freifunk.net'}
       },
 
+      wireguard = {
+        peers = {
+          vpn1 = {
+            public_key = 'XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX=',
+            endpoint = 'vpn1.alpha-centauri.freifunk.net:51810'
+          },
+          vpn2 = {
+            public_key = 'XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX=',
+            endpoint = 'vpn2.alpha-centauri.freifunk.net:51810'
+          }
+        }
+      },
```

Comment:
For consistency, always add trailing commas in tables:
```suggestion
      wireguard = {
        peers = {
          vpn1 = {
            public_key = 'XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX=',
            endpoint = 'vpn1.alpha-centauri.freifunk.net:51810',
          },
          vpn2 = {
            public_key = 'XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX=',
            endpoint = 'vpn2.alpha-centauri.freifunk.net:51810',
          },
        },
      },
```
While you're at it, you can also fix that in the `tunneldigger` section.